### PR TITLE
Docs: Fixed example in WebGLShader.html

### PR DESCRIPTION
**Pull request by jostschmithals**
WebGLShader (though upper case) is not designed to be used as a constructor.

`new THREE.WebGLShader` in the example wouldn’t work.

(+ Some additional adjustments)

### CODE REVIEW

**Comment by mrdoob** [comment]
It's not even in the THREE namespace, right? Be careful using those as we can't guarantee backward compatibility yet.

**Comment by jostschmithals** [comment]
Sorry, but I don't understand your answer(?) I suppose, it's a misunderstanding?

Different from what you wrote I didn't want to "use" something; with this PR I only wanted to remove some wrong information from the docs:

webglshader

replacing `new THREE.WebGLShader(...)` by `WebGLShader(...)`

(Though I didn't touch the whitespaces, the very few diffs can only be seen appending `&w=1` to the url:
https://github.com/mrdoob/three.js/pull/11031/commits/b638ab9389b8abed17b15614cf5cf86ab641f850?diff=unified&w=1)

But perhaps it would be better to remove the whole page from the docs?

**Comment by looeee** [comment]
> But perhaps it would be better to remove the whole WebGLShader-page from the docs

If I recall the idea was to move it to the wiki at some point and use that as a "developer reference" - however given that it would take quite a bit of work to do that well, and also that there is not really a clear boundary with regards to where this page falls, we decided to postpone that for the time being. 

And the page is useful enough that it should be kept *somewhere* in the meantime.

**Comment by jostschmithals** [comment]
@looeee I asked the quoted question not at all as a suggestion of mine to delete the page, but only because it was not clear to me how @mrdoob had meant his comment.

**Comment by mrdoob** [comment]
What I meant is that `WebGLShader` is not supposed to be "accessible".

screen shot 2017-03-21 at 16 22 26

It's basically not public because the API may still change. Not adding it to `THREE` used to do the trick for this, but now with the modules thing we no longer have control...

**Comment by pailhead** [comment]
@mrdoob

what are the benefits of not exporting parts of the renderer? 

Before, it was straight forward to monkey patch functionality to the renderer. Now it's impossible.

**Comment by mrdoob** [comment]
> what are the benefits of not exporting parts of the renderer?

Breaking less code.
